src/sudoers/test/mod.rs

```diff
@@ -20,16 +20,16 @@ impl UnixUser for Named {
         self.0 == name
     }
 
-    fn has_uid(&self, uid: u32) -> bool {
-        dummy_cksum(self.0) == uid
+    fn has_uid(&self, uid: UserId) -> bool {
+        dummy_cksum(self.0) == uid.get()
```

Comment:
The only way I could see this work is if we gave `GroupId` and `UserId` a similar trait (something that defines that they both have a unique identifier and can be root). But I'm not sure how to do that without overcomplicating it.

For now I changed the methods in test to compare two newtypes instead of calling `get`

Reply:
Maybe have two functions? `dummy_cksum_user` and `dummy_cksum_group` which return `UserId` and `GroupId` respectively? It is perfectly fine to return a different value for the user id and group id anyway.

Reply:
I think it's fine to have something like `GroupId::new(dummy_chksum(name))` for now, it is only test code after all.